workers/unity/Packages/com.improbable.gdk.core/Subscriptions/Systems/WorkerFlagCallbackSystem.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Subscriptions
+{
+    [DisableAutoCreation]
+    public class WorkerFlagCallbackSystem : ComponentSystem
+    {
+        private WorkerFlagCallbackManager manager;
+
+        public ulong RegisterWorkerFlagChangeCallback(Action<string, string> callback)
+        {
+            return manager.RegisterCallback(callback);
+        }
+
+        public bool UnregisterWorkerFlagChangeCallback(ulong callbackKey)
+        {
+            return manager.UnregisterCallback(callbackKey);
+        }
+
+        internal void InvokeCallbacks()
+        {
+            manager.InvokeCallbacks();
+        }
+
+        protected override void OnCreateManager()
+        {
+            base.OnCreateManager();
+            Enabled = false;
+
+            manager = new WorkerFlagCallbackManager(World);
+        }
+
+        protected override void OnUpdate()
```

Comment:
Don't think this needs an override?

Reply:
It does - the `OnUpdate()` is abstract on the base class.